features/networking/egress-ip.feature

```diff
@@ -111,3 +111,69 @@ Feature: Egress IP related features
     Then the step should succeed
     And the output should not contain "<%= cb.valid_ip %>"
     """
+
+  # @author huirwang@redhat.com
+  # @case_id OCP-21812 OCP-15473
+  # @bug_id  1609112
+  @admin
+  @destructive
+  Scenario: Should remove the egressIP from the array if it was not being used
+    Given I store a random unused IP address from the reserved range to the clipboard
+    And evaluation of `IPAddr.new("<%= cb.valid_ip %>").to_i + 1 ` is stored in the :newipint clipboard
+    And evaluation of `IPAddr.new(<%= cb.newipint %>, Socket::AF_INET).to_s` is stored in the :newip clipboard
+
+    #Patch egress cidr to the node
+    Given as admin I successfully merge patch resource "hostsubnet/<%= node.name %>" with:
+      | {"egressCIDRs": ["<%= cb.subnet_range %>"] }   |
```

Comment:
no need step ` the step should succeed`?

Reply:
Actually in the step definition, it has check about success or not:
 if ! success
    raise "patch failed to apply at #{failpath}! #{@result[:response]}"
  end

I checked this kind of step in all other cases, no additional step "the step should succeed" after it.

Reply:
ok, then LGTM, thanks